SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -92,7 +92,7 @@ namespace SpatialConstants
 		INVALID_ENTITY_ID = 0,
 		INITIAL_SPAWNER_ENTITY_ID = 1,
 		INITIAL_GLOBAL_STATE_MANAGER_ENTITY_ID = 2,
-		INITIAL_VIRTUAL_WORKER_TRANSLATOR_ENTITY_ID = 3,
+		INITIAL_VIRTUAL_WORKER_TRANSLATOR_ENTITY_ID = INITIAL_GLOBAL_STATE_MANAGER_ENTITY_ID,
```

Comment:
If we leave this on the gsm we need to make sure that we make the fields transient on the translator component and we clear then when in PIE. My preferred solution was to put this on another entity that isn't persistent. Then we can cleanup the ready state on the translator too.

Reply:
The prototype had it on a separate entity as per the original discussion.  Alex do you know why this was moved to GSM?

Reply:
It was on the GSM to be able to test the initialisation code quickly. There was then a discussion over whether or not it needs to be in the snapshot to which my preference is no as it's data that only makes sense for a specific instance of a deployment.

Reply:
I don't really think it belongs on the GSM, so if there's consensus I'd be happy to remove it, but I'll need someone to walk through the changes needed, since I don't have as much context on the initialization pieces.

Reply:
Looks like we all agree it shouldn't be on the GSM. Should this be addressed in a different PR?

Reply:
Happy to do that either for this or as a follow up task if we want to get the demo working. Outlined thoughts in UNR-2213

Reply:
Could you also add a TODO referencing UNR-2213 here?

Reply:
done. 
I'll also look into moving it out of the GSM, but as a separate PR at this point.